Sources/Futures/BackendService+Futures.swift

```diff
@@ -23,4 +24,19 @@ extension BackendServiceProtocol {
         
         return promise.future
     }
+
+    /// Executes the Request, returning a Future when finished.
+    ///
+    /// - Parameters:
+    ///   - recoverable: The Request to be executed, and is eligible to be recovered from in the case of failure.
```

Comment:
Minor: The comment here refers to the parameter as `recoverable`, but it's named `request` in the method signature below.

Reply:
Fixed - changed the signature so it's easy to delineate between the two when dynamic dispatch is involved.